src-java/kilda-pce/src/test/java/org/openkilda/pce/AvailableNetworkFactoryTest.java

```diff
@@ -220,6 +220,167 @@ public void shouldBuildAvailableNetworkForFlowWithIgnoreBandwidthPaths() throws
         assertAvailableNetworkIsCorrect(isl, availableNetwork);
     }
 
+    @Test
+    public void shouldBuildAvailableNetworkReusingBandwidthFromYFlowAndReusePathResources() throws Exception {
+        String yFlowId = "y-flow-id";
+        Flow flow1 = Flow.builder()
+                .flowId("test-id")
+                .srcSwitch(Switch.builder().switchId(new SwitchId("1")).build())
```

Comment:
we have a lot of constants in this class so you can reuse them if you want. For example 
`SWITCH_ID_1`

Reply:
done